go/test/endtoend/vtorc/readtopologyinstance/main_test.go

```diff
@@ -104,8 +105,19 @@ func TestReadTopologyInstanceBufferable(t *testing.T) {
 	assert.Equal(t, primaryInstance.ReplicationIOThreadState, inst.ReplicationThreadStateNoThread)
 	assert.Equal(t, primaryInstance.ReplicationSQLThreadState, inst.ReplicationThreadStateNoThread)
 
-	// insert an errant GTID in the replica
-	_, err = utils.RunSQL(t, "insert into vt_insert_test(id, msg) values (10173, 'test 178342')", replica, "vt_ks")
+	// Insert an errant GTID in the replica
```

Comment:
We are changing the way to create an errant gtid, because replicas have super-read-only = "ON" now, right? Maybe it will be useful to add that to the comments as well.